### PR TITLE
fix: #1073Added onchange callback in stepInput props

### LINKED ISSUE #1073: impossible to read value from StepInput
Description
How can we read value from StepInput so we can use it in a form? 
the ref is not forwarded to the input and onChange doesn't called when pressing on buttons.

Versions
0.10.0

### PR DESCRIPTION
**Pull request by athulvraj**
### Description
Added onchange callback in stepInput props

fixes #1073

### CODE REVIEW

File: src/StepInput/StepInput.js [hunk 1]
```diff
@@ -60,22 +61,26 @@ const StepInput = React.forwardRef(({
             return;
         }
         updateInputValue(inputValue - 1);
+        onChange(inputValue - 1);
```
[the remark this answers is not in the document]
Comment (athulvraj) [reply src/StepInput/StepInput.js]: done

**Comment by prsdthkr** [comment]
Wondering if `onChange` should be triggered, when non numeric keys are pressed (i.e. alphabets), even when the value doesn't change? Should there be some comparison with previous value?

**Comment by athulvraj** [comment]
> Wondering if `onChange` should be triggered, when non numeric keys are pressed (i.e. alphabets), even when the value doesn't change? Should there be some comparison with previous value?
> 

We don't support typing non digit characters in the input so when its pressed we won't trigger the onChange event

**Comment by prsdthkr** [comment]
It is currently being triggered as seen in the gif. Sorry didn’t make that clear.

Sent from my iPhone

On Jun 23, 2020, at 12:01 AM, athulvraj <notifications@github.com> wrote:

﻿

Wondering if onChange should be triggered, when non numeric keys are pressed (i.e. alphabets), even when the value doesn't change? Should there be some comparison with previous value?
[2020-06-22 10-53-20 2020-06-22 11_15_11]<https://user-images.githubusercontent.com/43764832/85321810-27810c80-b47a-11ea-9bc5-b28c04ff8086.gif>

We don't support typing non digit characters in the input so when its pressed we won't trigger the onChange event

**Pull request merged**